seisbench/generate/labeling.py

```diff
@@ -131,16 +131,19 @@ class PickLabeller(SupervisedLabeller, ABC):
     :param kwargs: Kwargs are passed to the SupervisedLabeller superclass
```

Comment:
Please add a docstring for `model_label`. And as I've just spotted that the one for `noise_column` is missing, would you mind adding that too?